### PR TITLE
Add file extensions

### PR DESCRIPTION
**Pull request by steveno**
* Also ensure that only the zig filetype plugin is loaded
  when opening a zig source file.

### CODE REVIEW

File: ftplugin/zig.vim [hunk 1]
```diff
@@ -1,5 +1,13 @@
...
+  finish
+endif
+
+let b:did_ftplugin = 1
+
 set expandtab
 set tabstop=4
 set shiftwidth=4
 
+setlocal suffixesadd=.zip
```
Comment (andrewrk) [inline ftplugin/zig.vim]: typo?

**Comment by andrewrk** [comment]
Isn't this what ftdetect/zig.vim is for? Or is this another thing?

File: ftplugin/zig.vim (hunk 1 above)
Comment (steveno) [reply ftplugin/zig.vim]: Yes.

**Comment by steveno** [comment]
It's different.

http://vimdoc.sourceforge.net/htmldoc/options.html#'suffixesadd'

**Comment by andrewrk** [comment]
thanks!

**Pull request merged**